### PR TITLE
Pay to person

### PR DESCRIPTION
**Pull request by Bongz12**

### CODE REVIEW

File: src/Components/WalletToWallet/WalletToWalletActions.js [hunk 1]
```diff
@@ -113,47 +121,48 @@ export const changePaymentUserInfo = (userInfo) => {
...
-    try {
-        const userData = onProcessPayment(cellNo);
-        dispatch(changeCellNoSuccess(userData));
-        return true;
-    } catch (e) {
-        dispatch(changeCellNoFailure(e));
-    }
+export const qrScan = ( objInfo ) => (dispatch) => {
+    dispatch(changeScanStart());
+    if ( objInfo.Lefa===true){
```
Comment (GeromeSchutte) [inline src/Components/WalletToWallet/WalletToWalletActions.js]: can just be `if (objInfo.Lefa) {`

Comment (Bongz12) [reply src/Components/WalletToWallet/WalletToWalletActions.js]: Why is this the case?

Comment (GeromeSchutte) [reply src/Components/WalletToWallet/WalletToWalletActions.js]: if () takes a boolean expression. Lefa can either be true, or undefined. true is a boolean expression, undefined evaluates to false if used as a boolean expression.

so youre essentially saying true === true, which is true, or false === true, which is false

Comment (Bongz12) [reply src/Components/WalletToWallet/WalletToWalletActions.js]: i cant believe you saw my stupid question..I read the change request wrong

**Pull request merged**